### ISSUE TITLE
Valid accessibility issues caught by smoketests

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#292665]
The to do clear button lacked an aria label and was nested within an element vs the container.
```

  1) VSCode Smoke Tests (Electron)
       Accessibility
         Chat
           chat response has no accessibility violations:
     Error: Accessibility violations found:

[aria-command-name] ARIA commands must have an accessible name (serious)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/aria-command-name?application=axeAPI
  Element: .codicon-clear-all
    Class: monaco-button codicon codicon-clear-all
    HTML: <a class="monaco-button codicon codicon-clear-all" tabindex="0" role="button" aria-disabled="false" custom-hover="true"></a>
    Issue: Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

[nested-interactive] Interactive controls must not be nested (serious)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/nested-interactive?application=axeAPI
  Element: a[aria-controls="todo-list-container"]
    Class: monaco-button
    HTML: <a class="monaco-button" tabindex="0" role="button" aria-disabled="false" aria-expanded="false" aria-controls="todo-list-container" aria-label="Expand Todos">
    Issue: Fix any of the following:
  Element has focusable descendants

Total: 2 violation(s) affecting 2 element(s)
      at Proxy.assertNoAccessibilityViolations (/Users/cloudtest/vss/_work/1/s/test/automation/out/playwrightDriver.js:628:19)
      at async Context.<anonymous> (out/areas/accessibility/accessibility.test.js:91:21)

```
```
  2) VSCode Smoke Tests (Electron)
       Accessibility
         Chat
           chat terminal tool response has no accessibility violations:
     Error: Accessibility violations found:

[aria-command-name] ARIA commands must have an accessible name (serious)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/aria-command-name?application=axeAPI
  Element: .codicon-clear-all
    Class: monaco-button codicon codicon-clear-all
    HTML: <a class="monaco-button codicon codicon-clear-all" tabindex="0" role="button" aria-disabled="false" custom-hover="true"></a>
    Issue: Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
[nested-interactive] Interactive controls must not be nested (serious)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/nested-interactive?application=axeAPI
  Element: a[aria-controls="todo-list-container"]
    Class: monaco-button
    HTML: <a class="monaco-button" tabindex="0" role="button" aria-disabled="false" aria-expanded="false" aria-controls="todo-list-container" aria-label="Expand Todos">
    Issue: Fix any of the following:
  Element has focusable descendants

Total: 2 violation(s) affecting 2 element(s)
      at Proxy.assertNoAccessibilityViolations (/Users/cloudtest/vss/_work/1/s/test/automation/out/playwrightDriver.js:628:19)
      at async Context.<anonymous> (out/areas/accessibility/accessibility.test.js:120:21)

```
